### ISSUE TITLE
Cannot modify header information - headers already sent by

### ISSUE THREAD

**Issue by Tahrz** [skipperbent/simple-php-router#429]
Hi! I have problem when set as callback Action with ```invoke()``` method.

```
Router::group(['namespace' => '\App\Actions\Tweets'], function () {
    Router::post('/feed', 'ListAction');
});
```
and in browser I see:

```
Notice: Undefined offset: 1 in /www/vendor/pecee/simple-router/src/Pecee/SimpleRouter/Route/Route.php on line 113
```

```
Warning: Cannot modify header information - headers already sent by (output started at /www/vendor/pecee/simple-router/src/Pecee/SimpleRouter/Route/Route.php:113) in /www/vendor/pecee/simple-router/src/Pecee/Http/Response.php on line 111
```
so...  in /www/vendor/pecee/simple-router/src/Pecee/SimpleRouter/Route/Route.php on line 113 we see:

```
$method = $controller[1];
```

And yes, i don`t use ```<controller name>@<method name>``` 

in my case it is ```<action name with __invoke method>```

 ... so  ```$controller[1]``` is empty.

**Comment by Tahrz** [comment]
so... simple spike fix:

```
if (sizeof($controller) === 1) {
    $controller[1] = '__invoke';
}

$method = $controller[1];
```

**Comment by skipperbent** [comment]
Hi @Tahrz.

Great work on resolving the issue, I'll make sure to include it in the next upcoming release.

Thanks for using the library.

/ Simon

**Comment by skipperbent** [comment]
The fix is available in version 4.2.0.6.

/ Simon
